python/config_settings/transition.bzl

```diff
@@ -43,24 +43,51 @@ def _transition_py_impl(ctx):
         output = executable,
         target_file = target[DefaultInfo].files_to_run.executable,
     )
-    zipfile_symlink = None
+    symlinks = []
     if target_is_windows:
-        # Under Windows, the expected "<name>.zip" does not exist, so we have to
-        # create the symlink ourselves to achieve the same behaviour as in macOS
-        # and Linux.
-        zipfile = None
-        expected_target_path = target[DefaultInfo].files_to_run.executable.short_path[:-4] + ".zip"
-        for file in target[DefaultInfo].default_runfiles.files.to_list():
-            if file.short_path == expected_target_path:
-                zipfile = file
+        files = target[DefaultInfo].files.to_list()
```

Comment:
I think the logic is sound, but its hard to follow because (1) symlinks is a list (I kept looking for where the second element is being added), and (2) the very similar logic has me playing "spot the difference", which is just a couple characters on the RHS in a couple places (which makes me think I'm missing something).

Can you clean up the logic to reduce the duplication? I think the two branches are identical except for:
* The suffix appended in declare_file()
* The executable=True/False value in symlink()

So it can be refactored to something like:
```
if build_zip:
  suffix = ".zip"
  executable = True
else:
  suffix = ""
  executable = False

launched_file = try_get_file_from_path(...)
launched_file_symlink = declare_file(...)
```